libecho/src/AudioStream.h

```diff
@@ -0,0 +1,127 @@
+#ifndef ECHOCONNECT_AUDIOSTREAM_H
+#define ECHOCONNECT_AUDIOSTREAM_H
+
+#include <QtCore/QThread>
+#include <QtMultimedia/QAudioFormat>
+#include <QtMultimedia/QAudio>
+#include <QtMultimedia/QAudioInput>
+#include <QDebug>
+#include <QMetaMethod>
+
+constexpr int NOTIFY_INTERVAL = 32;
```

Comment:
Let's try to move these constants somewhere else to be more flexible (or different for input and output).

Reply:
I don't heave and idea how to do it without putting them as class members (like I did with format). Help would be appreciated.

Reply:
Done